src/components/StatsInnerList.tsx

```diff
@@ -52,6 +53,14 @@ export default class StatsInnerList extends React.Component<StatsInnerListProps,
     return <ul className="stats-inner-list">{list}</ul>;
   }
 
+  getMonth(library: LibraryData): string {
```

Comment:
Thinking way ahead, this might be nice as a util function if it's ever needed outside this component.